src/Sentry.Serilog/SentrySink.cs

```diff
@@ -67,6 +78,15 @@ public void Emit(LogEvent logEvent)
         }
     }
 
+    private bool IsEnabled(LogEvent logEvent)
+    {
+        var options = _hubAccessor()?.GetSentryOptions();
```

Comment:
```suggestion
        var options = _hubAccessor().GetSentryOptions();
```

_hubAccessor() is never null

Reply:
I haven't changed to logic about null-checking of the `IHub` ... just continued the existing pattern.
I'll review if this is a necessary null-check, or a redundant leftover.